tests/src/test/java/org/wildfly/plugins/bootablejar/maven/goals/AbstractBootableJarMojoTestCase.java

```diff
@@ -118,7 +118,7 @@ public void shutdownServer() throws Exception {
         } finally {
             //Delete the build artifact dir
             Path buildArtifacts = getTestDir().resolve("target").resolve("bootable-jar-build-artifacts/");
-            BuildBootableJarMojo.deleteDir(buildArtifacts);
+            //BuildBootableJarMojo.deleteDir(buildArtifacts);
```

Comment:
Is this intentional or was it just for testing?

Reply:
Good catch...Thank-you!